modules/internal/ChapelArray.chpl

```diff
@@ -1942,6 +1942,25 @@ module ChapelArray {
     }
   }
 
+  @unstable("array casts are a new feature, seeking feedback and experience")
```

Comment:
Can we instead have this say:
```suggestion
  @unstable("Array-to-array casts are unstable and may change in the future. This feature is new and open to feedback.")  
```

Unstable warnings on the command line are not always clear that they are "unstable". Being verbose here helps the user know what kind of warning it is and also helps the `unstableWarningAnonymizer` script pick it up automatically as long as it has the word "unstable" in it.